src/deploy/scope_resourcegroup.ts

```diff
@@ -36,11 +36,13 @@ export async function DeployResourceGroupScope(azPath: string, resourceGroupName
             stderr: (data: BufferSource) => {
                 core.error(data.toString());
             },
-            stdline: (data: string) => {
-                if (!data.startsWith("[command]"))
```

Comment:
The intention was to not print the command which gets executed. That's why a line starting with `[command]` i being ignored.
How do we achieve the same now? Or is the command arguments masked already. In that case, it would be okay to do nothing.

Reply:
The debug logging is different for github actions and pipeline tasks. In tasks, the child process inherits the io streams of the parent process(called pipelining), so the child process will write the output to the main process streams. Only if explicitly specified in the exec options, can these streams be changed.
In actions, the toollib does not perform pipelining, and it exposes listeners to collect the child process streams. If we want to collect the library output/errors etc, we need to specify the streams in the options, else it defaults to the child process streams, which are't collected anywhere.